core/modules/field/modules/options/css/options.css

```diff
@@ -0,0 +1,102 @@
+
+div.options-widget .draggable a.tabledrag-handle {
+  padding-right: .3em /* RTL */
+}
+
+div.form-options div.options {
+  margin-bottom: 0;
+}
+
+div.options-widget a.add span,
+div.options-widget a.remove span {
+  display: none;
+}
+
+div.options-widget a.add,
+div.options-widget a.remove,
+div.form-option-add a {
+  display: block;
+  width: 16px;
+  height: 16px;
+  margin: .2em .1em;
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  float: left; /* RTL */
+}
+
+div.options-widget a.add:hover,
+div.options-widget a.remove:hover,
+div.form-option-add a:hover {
+  background-position: 0 -16px;
+}
+
+div.options-widget a.add {
+  background-image: url('../images/add.png');
+}
+
+div.options-widget a.remove {
+  background-image: url('../images/delete.png');
+}
+
+div.default-value-pattern-match {
+  float: left; /* RTL */
+}
+
+div.remove-default {
+  float: left; /* RTL */
+}
+
+div.form-options-manual,
+div.form-option-add {
+  text-align: right; /* RTL */
```

Comment:
Looks like these RTL styles are missing (difference from D7 code style). We should now add matching selectors that are prefixed like:

```
[dir="rtl"] .form-option-add {

}
```

Reply:
fixed

Reply:
I followed the field_ui.css for examples of how its done in core.